<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lesson 2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Task 1</string>
       </attribute>
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>391</width>
          <height>241</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Task 2</string>
       </attribute>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>6</x>
          <y>16</y>
          <width>274</width>
          <height>206</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QListView" name="listView"/>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="btnUp">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>20</y>
          <width>131</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Выше</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnDown">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>40</y>
          <width>131</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Ниже</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnDelete">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>60</y>
          <width>131</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="edLangToAdd">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>90</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btnAdd">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>110</y>
          <width>131</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="chkbOnlyIcons">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>140</y>
          <width>141</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Показать только иконки</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Task 3</string>
       </attribute>
       <widget class="QTableView" name="tblvComps">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>431</width>
          <height>251</height>
         </rect>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
       <widget class="QPushButton" name="btnFillColor">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>20</y>
          <width>80</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Закрасить</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>556</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
